fix: Doesn't disable X.509 in the project when mongodbatlas_x509_authentication_database_user resource is deleted (#1760)
mongodb/terraform-provider-mongodbatlas: website/docs/r/x509_authentication_database_user.html.markdown

-> **NOTE:** Groups and projects are synonymous terms. You may find group_id in the official documentation.

> -> **NOTE:** Before provider version 1.12.0, Self-managed X.509 Authentication was disabled for the project when this resource was deleted.

Reviewer: ```suggestion
-> **NOTE:** Before provider version 1.13.1, Self-managed X.509 Authentication was disabled for the project when this resource was deleted.
```

Author: sorry i meant to say before 1.14.0

> -> **NOTE:** Before provider version 1.14.0, Self-managed X.509 Authentication was disabled for the project when this resource was deleted.

Reviewer 2: This feels incomplete. Should we clarify that now we do nothing and why?

Author: thx, added more info

> -> **NOTE:** Before provider version 1.14.0, Self-managed X.509 Authentication was disabled for the project when this resource was deleted. From that version on, it is not disabled because the other users may want to continue using X.509.
> 

Reviewer 2: ```suggestion
-> **NOTE:** Before provider version 1.14.0, Self-managed X.509 Authentication was disabled for the project when this resource was deleted. From that version on, it won't be disabled to let other users continue using X.509 within the same project.

```

@lantoli check if `within the same project` is correct please. If you are ok, accept the suggestion (just a minor word refactoring).

Author: thx, changing for a similar sentence
